README.md

```diff
@@ -49,6 +49,21 @@ Launch the OpenFin runtime:
 npm run openfin
 ```
 
+### NPM Tasks
+
```

Comment:
General point - if you're in-lining the code, single back-ticks around the code is sufficient, e.g.

    `npm start`

Looks like:

`npm start`

----
Otherwise, if you want fenced code blocks, then 3 back-ticks surrounding the code is the way to go:

    ```
    npm start
    ```

Looks like:

```
npm start
```